api/modites.js

```diff
@@ -30,7 +50,13 @@ const getModites = async () => {
 
   // only ussers who are not bots, not restricted (usually clients) and not deleted
   const activeUsers = users.members.filter(user => user.name !== 'slackbot' && !user.is_bot && !user.is_restricted && !user.deleted);
+  const activeModitesList = activeUsers.map(item => item.id);
+  await MODITES.put('activeModitesList', JSON.stringify(activeModitesList));
```

Comment:
this `await` statements happen in series, meaning the next one will wait for the current to complete. Are they dependent on one another? Would it make sense to wrap some of them in a `Promise.all` block so they execute simultaneously?

Reply:
fixed